routes/tasks.js

```diff
@@ -134,28 +134,28 @@ router.put('/:id(\\d+)', csrfProtection, validateEditTask, asyncHandler( async (
 
         const validatorErrors = validationResult(req);
         if(validatorErrors.isEmpty()) {
-        const {title, estimate, listId, dueDate} = req.body;
-        await task.update({title, estimate, listId, dueDate})
-        // task.title = title;
-        // task.estimate = estimate;
-        // task.dueDate = dueDate
-        // if (listId) {
-        //     task.listId = listId
-        // }
-        res.status(201).json({task});
-        //TODO Implement AJAX
+            const {title, estimate, listId, dueDate} = req.body;
+            await task.update({title, estimate, listId, dueDate})
+            // task.title = title;
+            // task.estimate = estimate;
+            // task.dueDate = dueDate
+            // if (listId) {
+            //     task.listId = listId
+            // }
+            res.status(201).json({task});
+            //TODO Implement AJAX
 
         } else {
             const errors = validatorErrors.array().map((error) => error.msg);
-            console.error(errors);  
+            console.error(errors);
         }
     } else {
         next(taskNotFoundError(taskId))
     }
-    
+
 }));
 
-router.delete('/:id(\\d+)', validateTask, asyncHandler( async (req,res, next) => {
+router.post('/delete/:id(\\d+)', validateTask, asyncHandler( async (req,res, next) => {
```

Comment:
Don't forget to change to 'delete' once we figure out how it will work on the front end